tests/integration/team_test.py

```diff
@@ -17,34 +17,123 @@
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 #
 
+from contextlib import contextmanager
+import os
+
 import pytest
 
 import libnmstate
-from libnmstate.error import NmstateNotImplementedError
+from libnmstate.error import NmstateLibnmError
 from libnmstate.schema import Interface
+from libnmstate.schema import InterfaceState
 from libnmstate.schema import InterfaceType
 from libnmstate.schema import Team
 
+from .testlib import assertlib
+from .testlib import cmd as libcmd
+
+
+DNF_DOWNLOAD_NM_TEAM_PLUGIN_CMD = (
+    "dnf",
+    "install",
+    "-y",
+    "--downloadonly",
+    "NetworkManager-team",
+)
```

Comment:
Can we move this to `conftest.py` and make it run before all the test cases where network still works?

Reply:
I am fine with moving it to conftest.py. I will also send a PR to fix the OVS missing plugin test.

Reply:
Done, thanks! :-)